xla/service/gpu/cudnn_workspace_rewriter.cc

```diff
@@ -225,6 +225,56 @@ absl::StatusOr<se::gpu::CudnnGraph> HloCustomCallToCuDnnGraph(
             fmha_config.bias != std::nullopt, dnn_mask_type,
             force_deterministic));
     return std::move(graph);
+  } else if (IsFwdCustomCallTofMHAF8(*custom_call)) {
+    TF_ASSIGN_OR_RETURN(const xla::gpu::CudnnfMHAKind kind,
+                        xla::gpu::GetCudnnfMHAKind(custom_call));
+
+    TF_ASSIGN_OR_RETURN(
+        const auto gpu_config,
+        custom_call->backend_config<xla::gpu::GpuBackendConfig>());
+    const xla::gpu::CudnnfMHABackendConfig& config =
+        gpu_config.cudnn_fmha_backend_config();
+    Shape intermediate_tensor_shape(config.intermediate_tensor_shape());
+    absl::InlinedVector<Shape, 4> output_shapes = {
```

Comment:
Why does this have size 4?

Reply:
The outputs of fp8 call are: O, amax_s, amax_o and activation(attn score).